### ISSUE TITLE
Le bouton "Recommencer" devrait créer un nouvelle simulation dans la liste si un persona est sélectionné

### ISSUE THREAD

**Issue by Clemog** [incubateur-ademe/nosgestesclimat-site#1250]
Aujourd'hui on a un bug assez important dans le cas d'utilisation d'un persona:

Lorsque l'on sélectionne un persona et que l'on souhaite 'recommencer' une simulation sans persona, on se retrouve toujours avec ce persona sélectionné : on devrait donc mettre à jour `currentSimulationId` pour ne plus avoir cet identifiant dans le State.

Bug introduit depuis la mise en place d'une [liste de simulations sauvegardées](https://github.com/incubateur-ademe/nosgestesclimat-site/blob/master/source/sites/publicodes/SimulationList.tsx) dans le profil

**Comment by dxb** [comment]
J'ai l'impression qu'il n y'a que l'id du persona qui est conservé, il suffit peut être de générer un nouvel id lors de l'évènement reset_simulation.
Ou tout simplement, d'utiliser l'action deleteSimulationById à la place des actions du reset :
`	dispatch(resetSimulation())
	dispatch(resetActionChoices())
	dispatch(resetStoredTrajets())
	dispatch(resetStoredAmortissementAvion())
	dispatch(resetCategoryTutorials())`
Ca permettrait de se débarrasser 	de ses actions et de nettoyer un peu le rootReducer...

**Comment by margauxcoco** [comment]
De mon côté, j'ai plusieurs retours utilisateurs qui me demandent comment faire pour refaire le test tout en conservant les données de leur premier. Si on pouvait avoir un historique de simulations, ce serait très utile je pense.

**Comment by bjlaa** [comment]
> De mon côté, j'ai plusieurs retours utilisateurs qui me demandent comment faire pour refaire le test tout en conservant les données de leur premier. Si on pouvait avoir un historique de simulations, ce serait très utile je pense.

@dxb c'est pas ce qui est déjà en place ? Je viens d'essayer de faire une deuxième simulation et effectivement je n'ai pas accès à ma précédente simulation, pour parler en termes techniques : l'objet simulation est simplement vidé de sa situation si je comprends bien ?

En tout cas, je me demande si l'on ne pourrait pas revoir cela pour permettre la sauvegarde de plusieurs simulations et du coup peut-être changer l'intitulé du bouton "Recommencer" en "Repasser le test" ou autre avec une explication sur le fait que le précédent test peut être retrouvé à tout moment

**Comment by Clemog** [comment]
> @dxb c'est pas ce qui est déjà en place ? Je viens d'essayer de faire une deuxième simulation et effectivement je n'ai pas accès à ma précédente simulation, pour parler en termes techniques : l'objet simulation est simplement vidé de sa situation si je comprends bien ?

Nope, justement c'est seulement pour les personas de mémoire ! Effectivement, on le vide.. J'avais commencé qque chose [ici](https://github.com/incubateur-ademe/nosgestesclimat-site/commit/1607066775cb027e9da0f4570a8f9e6a1e743f81) mais je pense qu'il y a mieux à faire, notamment quand je regarde les points de ton [commentaire](https://github.com/incubateur-ademe/nosgestesclimat-site/issues/1258#issuecomment-1657998433)
